WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginActivity.java

```diff
@@ -158,14 +164,6 @@ protected void onCreate(Bundle savedInstanceState) {
 
     @Override
     public void onSaveInstanceState(Bundle outState) {
-        SignupGoogleFragment signupGoogleFragment;
```

Comment:
Following flow is fixed here -> this change requires to change a lot of other code - > mainly the `finishSignupFlow` needs to be invoked in all scenarios when the flow is finished.
```
Broken flow - happening mostly on low cost devices with limited memory (Samsung Galaxy Lite 2 - Android 4.4).
Click on "Sign Up With Google"
Account chooser is shown
Select an account
4a. Sometimes an infinite dialog which can't be canceled is shown
4b. Sometimes the dialog disappears, but nothing else happens (I think the activity's onSaveInstanceState gets called -> the GoogleSignUp fragment is removed)
```

Reply:
I think following flow got fixed as part of this change as well. It sometimes happens in the production version of the app but never happened in this branch. What might have also helped to fix this issue is, that we always disconnect the GoogleApiClient when the activityResult is returned.
```
Broken flow - weird state which I'm not sure how to reproduce, but happens from time to time (happened on both Pixel 2, Android 9 and Samsung Galaxy Lite 2, Android 4.4 phones)

Clic

1. k on "Sign Up with Google"
2. Semi-transparent grey overlay is shown
3. Nothing else happens
```